scala/build/maintenance.Dockerfile

```
@@ -45,7 +45,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     libgstreamer-plugins-bad1.0-0 \
     gstreamer1.0-plugins-good \
     gstreamer1.0-libav
-
+t
```

Hadolint

[error] unexpected 't'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)